src/main/groovy/org/mockito/release/internal/gradle/DefaultReleaseNotesPlugin.java

```diff
@@ -56,15 +57,29 @@ public void execute(Task task) {
             }
         });
 
+        project.getTasks().create("fetchNotableNotes", NotableNotesFetcherTask.class, new Action<NotableNotesFetcherTask>() {
+            public void execute(NotableNotesFetcherTask task) {
+                final NotesGeneration gen = task.getNotesGeneration();
+                preconfigureNotableNotes(project, gen);
+
+                task.getOutputs().file(getTemporaryReleaseNotesFile(project));
```

Comment:
I suggest to use Gradle annotations to drive the inputs/output, @OutputFile, @Input, etc. Check out Gradle docs on this :)